ui/common/css/abstract/_media-queries.scss

```diff
@@ -45,8 +45,8 @@ $mq-hover-no: hover none;
 
 $mq-main-margin: $mq-small;
 
-$mq-topnav-visible: $mq-medium;
-$mq-topnav-hidden: $mq-not-medium;
+$mq-topnav-visible: min-width (1022px);
+$mq-topnav-hidden: max-width (1022px); // for user WWWWWWWWWWWWWWWWWWWW at 100% font size
```

Comment:
media queries close to common screen sizes (such as 1024) should be avoided, it's better to aim in between screen sizes. If a browser has a 1px border on each size then this media query will flip.

Also why the new hardcoded values? Shouldn't we be restricting the username display size instead, if that's the problematic edge case

Reply:
The value 1022 was chosen as the point we start pushing long usernames off the screen. I wanted it below 1024 since that's tablet res, but it can come down a bit more sure